salt/master_utils/nodegroups.sls

```diff
@@ -0,0 +1,47 @@
+manage_nodegroups:
+  file.managed:
```

Comment:
This state isn't necessary. You can add the nodegroups definitions as YAML to the salt_master.sls pillar file under the `extra_configs` section (https://github.com/mitodl/salt-ops/blob/redash_scaling/pillar/salt_master.sls#L102). The key name becomes the filename and everything nested under that gets written to the associated configuration file on the salt master via the `master.config` state from the `master-formula` repository.

Reply:
OK, I've moved it.